readoutcard.sh

```diff
@@ -55,6 +55,7 @@ module load BASE/1.0                                                          \\
 setenv READOUTCARD_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
 prepend-path PATH \$::env(READOUTCARD_ROOT)/bin
 prepend-path LD_LIBRARY_PATH \$::env(READOUTCARD_ROOT)/lib
+prepend-path PYTHONPATH \$::env(READOUTCARD_ROOT)/lib
```

Comment:
Out of curiosity, is there any reason why Python modules for readout are installed in a rather non-standard path? Normally they will go in a `site-packages` or `dist-packages` directory or something (look at the `yoda.sh` recipe for instance).

Reply:
Well, there is actually no separate module. The libReadoutCard.so C++ library is also directly usable by Python.

Reply:
OK, but isn't `LD_LIBRARY_PATH` sufficient then? Pardon my ignorance!

Reply:
Unfortunately not! We need PYTHONPATH for the import statement to work